src/java/org/apache/poi/ss/formula/FormulaParser.java

```diff
@@ -271,7 +271,17 @@ private RuntimeException expected(String s) {
 
     /** Recognize an Alpha Character */
     private static boolean IsAlpha(int c) {
-        return Character.isLetter(c) || c == '$' || c=='_';
+        return IsLetter(c) || c == '$' || c=='_';
+    }
+
+    /** Recognize a Letter Character */
+    private static boolean IsLetter(int c) {
+        return Character.isLetter(c) || c >= 256;
```

Comment:
Won't this incorrectly catch special unicode characters such as an em space? Should whitespace be considered a letter here?
https://en.wikipedia.org/wiki/Whitespace_character